src/extensions/filter-control/utils.js

```diff
@@ -342,15 +340,16 @@ export function createControls (that, header) {
       }
     }
 
-    if (!column.filterControl && '' !== column.filterDefault && 'undefined' !== typeof column.filterDefault) {
+    // Filtering by default when it is set.
+    if (column.filterControl && '' !== column.filterDefault && 'undefined' !== typeof column.filterDefault) {
       if ($.isEmptyObject(that.filterColumnsPartial)) {
         that.filterColumnsPartial = {}
       }
 
       that.filterColumnsPartial[column.field] = column.filterDefault
     }
 
-    $.each(header.find('th'), (i, th) => {
+    $.each(header.find('th'), (_, th) => {
```

Comment:
`_` ?

Reply:
It is used when you are handle that parameters as 'Discard' one. See https://docs.microsoft.com/en-us/dotnet/csharp/discards

Reply:
Good to know! I already thought that, but i was not sure if im right.